style.css

```diff
@@ -82,3 +87,331 @@
   padding-left: 0;
   padding-top: 20px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  width: 375px;
+  height: 2788px;
+  background: #f7f7f9;
+}
+
```

Comment:
- [ ] According to requirement, this should have a `display: grid` not `flex`. Kindly check [the project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and make it as required.